source/utilities.tex

```diff
@@ -15012,24 +15056,27 @@
 \item \ucode{20000} -- \ucode{2fffd}
 \item \ucode{30000} -- \ucode{3fffd}
 \end{itemize}
-The estimated width of other code points is 1.
+The field width of all other code points is 1.
 
 \pnum
-For a string in a non-Unicode encoding, the width of a string is unspecified.
+For a sequence of characters in a non-Unicode encoding,
+the field width is unspecified.
 
 \pnum
-% FIXME: What if it's an arg-id?
-The \fmtgrammarterm{nonnegative-integer} in
-\fmtgrammarterm{precision} is a decimal integer defining
-the precision or maximum field size.
-It can only be used with floating-point and string types.
-For floating-point types this field specifies the formatting precision.
-For string types, this field provides an upper bound
-for the estimated width of the prefix of
-the input string that is copied into the output.
-For a string in a Unicode encoding,
-the formatter copies to the output
-the longest prefix of whole extended grapheme clusters
+The \fmtgrammarterm{precision} option is valid
+for floating-point and string types.
+For floating-point types,
+the value of this option specifies the precision
+to be used for the floating-point presentation type.
+For string types,
+this option specifies the longest prefix of the formatted argument
+to be included in the replacement field such that
+the field width of the prefix is no greater than the value of this option.
+
+\pnum
+If \fmtgrammarterm{nonnegative-integer} is used in
+a \fmtgrammarterm{precision} option,
+the value of the decimal integer is used as the value of the option.
 whose estimated width is no greater than the precision.
```

Comment:
This line is part of the deleted paragraph.
```suggestion
```

Reply:
Thanks, must not have gotten enough sleep before starting this one :)